### ISSUE TITLE
search by barcode

### ISSUE THREAD

**Issue by maxlath** [inventaire/inventaire-client#388]
we could borrow this nice UI idea from bookwyrm
![a654efe81e034482](https://user-images.githubusercontent.com/1596934/214135067-9cbe04a7-1001-4625-9d83-8a0824e582d2.png)

[screenshot source](https://mastodon.social/@blinry@chaos.social/109739930230777138)

**Comment by jum-s** [comment]
tldr: having several action buttons to distinguish "search for a book" and "import books to my inventory"

Makes sense since the ISBN search is, imo, missing some incentive in the global search bar. 

I see 2 possible implementations
- click on next, go to the current scan/importer process to scan multiple books. But i think the importer interface is a big cognitive thing to deal with, while you might just want to look for this one book.
- having to buttons "search for this on book" and "import books to your inventory", the second path could be as follow: stay in this view, open a modal, scan bar code, close the modal when scanner has found ISBN, display edition in the search result list.
